src/lib/storage/boot_requirements_strategies/base.rb

```diff
@@ -22,6 +22,13 @@
 # find current contact information at www.suse.com.
 
 require "yast"
+require "storage/refinements"
+require "storage/disk_size"
+require "storage/planned_volume"
+require "storage/planned_volumes_list"
+require "storage/proposal/exceptions"
```

Comment:
It looks strange that a class that lives out of the Proposal namespace contains this include.